tests/backfill/test_dir_multiple/backfill.yaml

```diff
@@ -0,0 +1,19 @@
+2023-05-03:
+  start_date: 2023-01-03
+  end_date: 2023-05-03
+  excluded_dates:
```

Comment:
It feels like a none standard thing to exclude dates from a backfill, maybe we should also include a note field for it to provide info why dates are being excluded.

Reply:
I'd recommend using the `reason` field here since it already exists